File: Pyramid.java
public class Pyramid {
    public static void main(String[] args) {
        StringBuilder sb = new StringBuilder();
        int rows = 15;
        int k = 0;
        for (int i = 1; i <= rows; i++, k = 0) {
            for (int space = 0; space <= rows - i; space++)
                sb.append(" ");
            while (k++ != i - 1)
                sb.append("* ");
            System.out.println(sb.toString());
            sb.setLength(0);
        }
    }
}

File: Pyramid.py
rows = int(input("Rows: "))
space = rows if rows % 2 else rows - 1
for i in range(rows):
    for j in range(space):
        print(end=" ")
    space -= 1
    for j in range(i + 1):
        print("*", end=" ")
    print("")
